composer.json

```diff
@@ -18,7 +18,8 @@
         "illuminate/config": "^6.0|^7.0|^8.0",
         "illuminate/database": "^6.0|^7.0|^8.0",
         "illuminate/support": "^6.0|^7.0|^8.0",
-        "illuminate/translation": "^6.0|^7.0|^8.0"
+        "illuminate/translation": "^6.0|^7.0|^8.0",
+        "phpunit/phpunit": "^9.5"
```

Comment:
Isn't phpunit meant to be in `require-dev` ?

Reply:
Ah, whoops! Forgot the `--dev` flag

Reply:
Moved it back to dev